clouddriver-saga/src/main/kotlin/com/netflix/spinnaker/clouddriver/saga/flow/SagaAction.kt

```diff
@@ -39,6 +39,17 @@ interface SagaAction<in T : SagaCommand> {
    */
   fun apply(command: T, saga: Saga): Result
 
+  /**
+   * In the event of an exception being raised from [apply], a [SagaAction] can implement custom error handling logic.
+   *
+   * By default, nothing happens.
+   *
+   * @param command The input [SagaCommand] that was acted on
+   * @param saga The [Saga] state used to apply the [command]
+   * @param exception The resulting exception
+   */
+//  fun recover(command: T, saga: Saga, exception: Exception): Result = Result()
```

Comment:
intended to be commented out?

Reply:
Yes, this behavior isn't written yet / haven't decided the specific contract yet.